### DISCUSSION TITLE
Netbeans 12.6 editor goes non-responsive - Mac OS Montery

### DISCUSSION THREAD

**Discussion by crashmeister** [apache/netbeans#3589] [Q&A]
Works fine when initially loaded. After switching apps (CMD-Tab) multiple times, when returning to Netbeans, there is no cursor and editing is effectively disabled. Menus are still accessible and work. 

This is very hard to reproduce. I have yet to detect an exact pattern. I use BBEdit and Netbeans at the same time and switch between them frequently. Other apps often used at the same time include Finder, Safari and Terminal.

**Comment by oyarzun** [comment]
@crashmeister I have run into this issue for a while as well. I'm on Catalina. I found that clicking on another application then back in the NetBeans editor will get the cursor back.

I've finally figured out how to consistently duplicate it.

1.  Use CMD-Tab to move a window in front of NetBeans
2. Right-Click on Project and perform an action like Clean
3. Go back to Editor and the cursor is gone.

![2022-02-18 08 24 10](https://user-images.githubusercontent.com/2364606/154691277-0742861b-2884-4afb-8bdf-32d14900364c.gif)
